docs/src/main/sphinx/release/release-360.md

```diff
@@ -0,0 +1,71 @@
+# Release 360 (XXX Jul 2021)
+
+## General
+
+* Reduce number of requests from coordinator to workers ({issue}`8460`)
+* Improve query performance by sending collected dynamic filters from coordinator to workers. ({issue}`5183`)
+* Improve performance of inequality joins where join condition sides have different type. ({issue}`8500`)
+* Fix query failure when there is a constant predicate on some
+  column `col` (e.g `col=1`), followed by  `ORDER BY col` and `LIMIT`. ({issue}`8535`)
+* Fix SHOW CREATE SCHEMA. ({issue}`8598`)
+* Fix query failure when running `SHOW CREATE SCHEMA information_schema`. ({issue}`8600`)
+* Improve performance of queries with `WHERE` predicates. ({issue}`8624`)
```

Comment:
that's very broad. can it be anyhow more specific? @sopel39

Reply:
Given the nature of that PR, it's not specific to any special filter predicate.

Reply:
Maybe:

> Improve performance of `WHERE` clause evaluation.